packages/apps/src/configs/evm/SupportedMixers.ts

```diff
@@ -0,0 +1,78 @@
+export enum WebbEVMChain {
+  Main = 1,
+  Rinkeby = 4,
+  Edgeware = 2021,
+  Beresheet = 2022,
+}
+
+export const getNativeCurrencySymbol = (chainID: number): string => {
+  switch (chainID) {
+    case WebbEVMChain.Rinkeby:
+      return 'ETH';
+    case WebbEVMChain.Main:
+      return 'ETH';
+    case WebbEVMChain.Beresheet:
+      return 'tEDG';
+    case WebbEVMChain.Edgeware:
+      return 'EDG';
+    default:
+      throw new Error('unsupported chain');
+  }
+}
```

Comment:
What would the code look like for this? 
Something like: 
```
    case WebbEVMChain.Rinkeby:
         return currenciesConfig[chainsConfig[3].nativeCurrencyId].symbol;
```

Since we do not have a clean mapping from the evm reported chainId and what we consider a chainId in the dapp's `chain-config.ts`, it seems this would require some hardcoding for the index of the chainConfig.

Reply:
yeah this would be fine but let's handle this on another PR
Great work :D